Pushover.indigoPlugin/Contents/Server Plugin/plugin.py

```diff
@@ -85,15 +88,17 @@ def send(self, pluginAction):
 
 		if self.present(pluginAction.props.get('msgAttachment')):
 			attachFile = self.prepareTextValue(pluginAction.props['msgAttachment'])
-			if os.path.isfile(attachFile) and attachFile.lower().endswith(('.jpg', '.jpeg')):
+			extension = os.path.splitext(attachFile)[1]
+
+			if os.path.isfile(attachFile) and (extension == '.jpg' or extension == '.jpeg' or extension == '.gif'):
```

Comment:
This is no longer lowercasing the filename, so it won't catch images with `.JPG` or similar extensions.

Reply:
fixed in 1.5.1

Reply:
Thanks guys!